sigmf/sigmffile.py

```diff
@@ -948,7 +948,8 @@ def get_dataset_filename_from_metadata(meta_fn, metadata=None):
     compliant_filename = get_sigmf_filenames(meta_fn)["data_fn"]
     noncompliant_filename = metadata["global"].get(SigMFFile.DATASET_KEY, None)
 
-    if path.isfile(compliant_filename):
+    if Path.is_file(compliant_filename):
+    # if path.isfile(compliant_filename):
```

Comment:
This PR is looking almost ready. You have a few commented lines leftover and I see lots of places where you just need some formatting like `a/ b` to `a / b`